docs/sources/configure-client/grafana-alloy/java/_index.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Profiling Java using Grafana Alloy"
-menuTitle: "Profiling Java using Alloy"
+title: "Set up Java profiling using Grafana Alloy"
+menuTitle: "Set up Java profiling"
 description: "Learn about using Grafana Alloy for continuous profiling Java processes for performance optimization."
```

Comment:
Maybe since we changed it above?
```suggestion
description: "Learn about using Grafana Alloy for continuous Java profiling processes for performance optimization."
```